static/app/components/idBadge/teamBadge/index.tsx

```diff
@@ -1,57 +1,40 @@
 import * as React from 'react';
-import isEqual from 'lodash/isEqual';
 
 import TeamStore from 'sentry/stores/teamStore';
 import {Team} from 'sentry/types';
 
-import Badge from './badge';
+import Badge, {BadgeProps} from './badge';
 
-type Props = React.ComponentProps<typeof Badge>;
+function TeamBadge(props: BadgeProps) {
+  const [team, setTeam] = React.useState<Team>(props.team);
```

Comment:
are you sure of adding the type Team here? I still find it verbose

Reply:
Probably a good topic for the TSC :)